packages/module/tests.spec.ts

```diff
@@ -0,0 +1,18 @@
+// import { refs, injectReferences } from "./index"
```

Comment:
remove or update :)

Reply:
I commented to update in next iteration (i want to merge it asap bc of conflicts) but If this hurts your eyes i can delete and copy from older commits